README.md

```diff
@@ -1,14 +1,35 @@
 ![Cover logo](./cover.svg)
 
-# instant-segment: fast English word segmentation in Rust
+# Instant Segment: fast English word segmentation in Rust
 
 [![Documentation](https://docs.rs/instant-segment/badge.svg)](https://docs.rs/instant-segment/)
 [![Crates.io](https://img.shields.io/crates/v/instant-segment.svg)](https://crates.io/crates/instant-segment)
+![PyPI](https://img.shields.io/pypi/v/instant-segment)
 [![Build status](https://github.com/InstantDomainSearch/instant-segment/workflows/CI/badge.svg)](https://github.com/InstantDomainSearch/instant-segment/actions?query=workflow%3ACI)
 [![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE-APACHE)
 
-instant-segment is a fast Apache-2.0 library for English word segmentation.
-It is based on the Python [wordsegment][python] project written by Grant Jenkins,
+```python
+segmenter = instant_segment.Segmenter(unigrams(), bigrams())
+search = instant_segment.Search()
+segmenter.segment("instantdomainsearch", search)
+print([word for word in search])
+
+--> ['instant', 'domain', 'search']
+```
+
+```rust
+let segmenter = Segmenter::from_maps(unigrams, bigrams);
```

Comment:
Here you directly use `Segmenter` and `Search` through the `instant_segment` module, whereas in the Rust code you use implicitly imported names. Maybe we should make both examples import these explicitly? I think that would be more idiomatic (in most cases) than qualifying the names, and clearer (especially for copy/pasting) than keeping the imports implicit.